dojson/utils.py

```diff
@@ -96,8 +96,12 @@ def wrapper(self, key, values, **kwargs):
 
 def force_list(data):
     """Wrap data in list."""
-    if data is not None and not isinstance(data, (list, tuple, set)):
-        return (data,)
+    if data is None:
+        return []
```

Comment:
> ... having the inelegant pattern

IMHO this pattern is very elegant. Please come back to https://github.com/inveniosoftware/dojson/pull/155/files#r70324927 , https://github.com/inveniosoftware/dojson/pull/155/files#r70324824 , and https://github.com/inveniosoftware/dojson/pull/155/files#r70324884